src/app/courses/create/page.tsx

```diff
@@ -0,0 +1,14 @@
+import CourseForm from '@/components/CourseForm/CourseForm';
+import { prisma } from '@/lib/prisma/prisma';
+
+export const dynamic = 'force-dynamic';
+
+export default async function NewCoursePage() {
+  const tags = await prisma.tag.findMany();
```

Comment:
```suggestion
  const tags = await prisma.tag.findMany({orderBy: {name: 'asc'}});
```

You probably want the tags in alphabetical order to make browsing them easier? The placeholder ones were just by coincidence in alphabetical order.

Reply:
Agreed; an overlook by me. Switched to using a helper function instead of querying the db in a component altogether. See cc34385